pkg/config/controller/config.go

```diff
@@ -3,9 +3,9 @@ package controller
 import "github.com/kubevirt/vm-import-operator/pkg/config"
 
 const (
-	osConfigMapNamespaceKey = "osConfigMap.namespace"
+	OsConfigMapNamespaceKey = "osConfigMap.namespace"
```

Comment:
would it require comment since it is exported ?

Reply:
Fixed.